data/pom.yml

```diff
@@ -11,70 +11,70 @@ kieExecutionServerDescription: Standalone execution server that can be used to r
+    businessCentralSnapWF: https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.kie&a=business-central&v=7.63.0-SNAPSHOT&e=war&c=wildfly19
+    kieserverSnapEe7: https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.kie.server&a=kie-server&v=7.63.0-SNAPSHOT&e=war&c=ee7
+    kieserverSnapEe8: https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.kie.server&a=kie-server&v=7.63.0-SNAPSHOT&e=war&c=ee8
+    kieserverSnapWebc: https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.kie.server&a=kie-server&v=7.63.0-SNAPSHOT&e=war&c=webc
+    kieserverSnapDistZip: https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.kie.server&a=kie-server-distribution&v=7.63.0-SNAPSHOT&e=zip
+    tomcatIntergrationSnap: https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.kie&a=kie-tomcat-integration&v=7.63.0-SNAPSHOT&e=jar
+    # updatesiteSnapZip: https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.drools&a=org.drools.updatesite&v=7.63.0-SNAPSHOT&e=zip
 
 # latest.version can be equal to latestFinal.version
 latest:
-    version: 7.61.0.Final
-    releaseDate: 2021-11-04
-    droolsZip: https://download.jboss.org/drools/release/7.61.0.Final/drools-distribution-7.61.0.Final.zip
+    version: 7.62.0.Final
+    releaseDate: 2021-11-29
+    droolsZip: https://download.jboss.org/drools/release/7.62.0.Final/drools-distribution-7.62.0.Final.zip
 
-    droolsjbpmIntegrationZip: https://download.jboss.org/drools/release/7.61.0.Final/droolsjbpm-integration-distribution-7.61.0.Final.zip
+    droolsjbpmIntegrationZip: https://download.jboss.org/drools/release/7.62.0.Final/droolsjbpm-integration-distribution-7.62.0.Final.zip
 
-    droolsWorkbenchEAP7WAR: https://download.jboss.org/drools/release/7.61.0.Final/business-central-7.61.0.Final-eap7.war
-    droolsWorkbenchWildFlyWAR: https://download.jboss.org/drools/release/7.61.0.Final/business-central-7.61.0.Final-wildfly23.war
-    droolsWorkbenchJcr2vfsZip: https://download.jboss.org/drools/release/7.61.0.Final/drools-wb-jcr2vfs-distribution-7.61.0.Final.zip
-    droolsWorkbenchExampleGitReposZip: https://download.jboss.org/drools/release/7.61.0.Final/kie-wb-example-repositories-7.61.0.Final.zip
+    droolsWorkbenchEAP7WAR: https://download.jboss.org/drools/release/7.62.0.Final/business-central-7.62.0.Final-eap7.war
+    droolsWorkbenchWildFlyWAR: https://download.jboss.org/drools/release/7.62.0.Final/business-central-7.62.0.Final-wildfly23.war
+    droolsWorkbenchJcr2vfsZip: https://download.jboss.org/drools/release/7.62.0.Final/drools-wb-jcr2vfs-distribution-7.62.0.Final.zip
+    droolsWorkbenchExampleGitReposZip: https://download.jboss.org/drools/release/7.62.0.Final/kie-wb-example-repositories-7.62.0.Final.zip
```

Comment:
i can't find this URL, is it still in released assemblies? I can't see it even in previous releases

Reply:
checking

Reply:
@mareknovotny a business-central-{$kie-version}-eap7.war is not any more provided - this for long time! But is is still in the the pom. You have not displayed this link in the drools webpage (www.drools.org).
Also in the latest - there still were not working links but also they didn't get displayed in the web. Have removed these.
Another thing is iof we still need the `latest` - sinc the links in the web are pointing to `latest.final`